README.md

```diff
@@ -77,6 +74,9 @@ The best and quickest way to get value out of wGulp is to adhere to a standard d
 * `./dist` - Destination for generated distribution assets
 * `./docs` - Destination for generated documentation
 * `./report` - Destination for generated reports (coverage, complexity, test)
+* `./sass` - Default location for OOCSS styles
+* `./src` - TypeScript, JavaScript, JSX, Coffee, ... source code
```

Comment:
Alphabetized and added more description to the src directory to make it clear that all other languages that need to be transpiled also live in src.